students/rfelts/lesson03/strformat_lab.py

```diff
@@ -0,0 +1,103 @@
+# Russell Felts
+# Assignment 3 - String Formatting Exercise
+
+
+# Step 1 & 2
+# Use string formatting to convert the tuple data to:
+# 'file_002 :   123.46, 1.00e+04, 1.23e+04'
+#
+# The first element is used to generate a filename. Pad the numbers with zeros
+# to get the right sort order.
+#
+# The second element is a float displayed with 2 decimal places.
+#
+# The third value is an integer displayed with in scientific notation, with 2
+# decimal places shown.
+#
+# The fourth value is a float displayed in scientific notation with 3
+# significant figures.
+
+
+tuple_1 = (2, 123.4567, 10000, 12345.67)
+
+
+formated_tuple = 'file_{:0>3d},{:.2f},{:.2e},{:.2e}'.format(*tuple_1)
+print('Step 1:\nFormatting the tuple:', tuple_1,
+      "\nUsing the format method.", formated_tuple, "\n")
+
+
+fstring_tuple = (f"file_{tuple_1[0]:03d}, {tuple_1[1]:.2f}," +
+                 f"{tuple_1[2]:.2e}, {tuple_1[3]:.2e}")
+print('Step 2:\nFormatting the tuple:', tuple_1,
+      "\nUsing f-String.", fstring_tuple, "\n")
```

Comment:
Good - assignment was to literally print out 'file_002 :   123.46, 1.00e+04, 1.23e+04'